rules/0605-win-mcafee_rules.xml

```diff
@@ -42,34 +42,34 @@
 
   <rule id="62603" level="2">
     <if_sid>62600</if_sid>
-    <field name="win.system.eventID">$MCAFEE_INFO</field>
+    <field name="win.system.eventID">$MCAFEE_INFO$</field>
```

Comment:
`$MCAFEE_INFO` is a variable defined as `<var name="MCAFEE_INFO">^257$|^5000$|^5026$|^5052$|^5055$</var>` so it is not needed to include the `$` character.